yt/visualization/_commons.py

```diff
@@ -44,9 +66,9 @@ def validate_image_name(filename, suffix: Optional[str] = None) -> str:
     suffix = normalize_extension_string(suffix)
 
     if suffix not in SUPPORTED_FORMATS:
-        raise ValueError("Unsupported file format '{suffix}'.")
+        raise ValueError(f"Unsupported file format '{suffix}'.")
```

Comment:
Nice catch!

Reply:
always nice to fix my own mistakes before I get caught :)